src/app/components/PortraitVideoCarousel/index.styles.ts

```diff
@@ -26,6 +26,9 @@ const styles = {
       position: 'relative',
       display: 'flex',
       alignItems: 'center',
+      '.no-js &': {
```

Comment:
We have a const value for the NO-JS classname:
`import NO_JS_CLASSNAME from '#app/lib/noJs.const';`

which can be used like:
```
      [`.${NO_JS_CLASSNAME} &`]: {
        display: 'none',
      },
```